tests/fixtures/verify-fleet-daytona.test.ts

```
@@ -0,0 +1,1754 @@
+      escapedPid = Number(result._rawStdout.trim());
+
+      expect(result.timedOut).toBe(true);
+      expect(result.durationMs).toBeLessThan(3_000);
+      expect(Number.isSafeInteger(escapedPid)).toBe(true);
+      expect(Date.now() - startedAt).toBeLessThan(3_000);
+    } finally {
+      if (escapedPid && Number.isSafeInteger(escapedPid)) {
+        try {
+          process.kill(escapedPid, 'SIGKILL');
+        } catch (error: any) {
+          if (error?.code !== 'ESRCH') throw error;
```

Biome (2.5.8)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)